azure/functions_worker/dispatcher.py

```diff
@@ -209,9 +209,14 @@ def _serialize_exception(self, exc):
     async def _handle__worker_init_request(self, req):
         logger.info('Received WorkerInitRequest, request ID %s',
                     self.request_id)
+
+        capabilities = dict()
+        capabilities["RawHttpBodyBytes"] = "true"
```

Comment:
`RawHttpBodyBytes` move this to constants. Overtime, there might be more capabilities